src/ui/common/src/components/integrations/dialogs/lambdaDialog.tsx

```diff
@@ -20,7 +20,7 @@ export const LambdaDialog: React.FC<Props> = ({ onUpdateField, value }) => {
         spellCheck={false}
         required={true}
         label="Lambda Role ARN"
-        description="Region in which to run Lambda functions."
+        description="ARN for Lambda executor role."
```

Comment:
Is there a guide we could link to? Either ours or Amazons? I see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html#roles-creatingrole-service-console

Reply:
or maybe its https://docs.aws.amazon.com/lambda/latest/dg/lambda-intro-execution-role.html

Reply:
Yea either our docs at https://github.com/aqueducthq/gitbook/blob/main/integrations/adding-an-integration/connecting-to-aws-lambda.md or the amazon link here: https://docs.aws.amazon.com/lambda/latest/dg/lambda-intro-execution-role.html#permissions-executionrole-console should work. I'm unsure why the documentation page isn't appearing directly on gitbook, i'll look into that.